fixes #15

alexeiva opened the pull request:
fixed interpolation issues in Cyrillic

alexeiva committed:
fixed interpolation issues in Cyrillic Raleway-Italic v4015. Fixes https://github.com/impallari/Raleway/issues/15. Did not regenerate fonts

alexeiva committed:
sync with Pablo

alexeiva committed:
fixed interpolation issues in Cyrillic

alexeiva committed:
regenerated fonts

alexeiva committed:
working on Cyr Pro

alexeiva committed:
worked on hooks ԒԓԨԩ

alexeiva committed:
backup

alexeiva committed:
working on Cyr Pro

alexeiva committed:
more work on CYR hooks

alexeiva committed:
reworked cyr hooks

alexeiva committed:
added ҩ

alexeiva committed:
Ҩҩ

alexeiva committed:
completed CYR Pro in Roman

alexeiva committed:
expanding Italic to GF CYR Pro

alexeiva committed:
working on Italic Cyr Pro

alexeiva committed:
renamed source files

alexeiva committed:
completed drawing GF CYR Pro Roman+Italic

alexeiva committed:
set metrics using HT Letterspacer

alexeiva committed:
updated copyright accodring to spec

alexeiva committed:
additional kerning for CYR Pro

alexeiva committed:
additional kerning for CYR Pro in Roman master

alexeiva committed:
generated fonts

alexeiva committed:
repo reorganisation

alexeiva committed:
updated README.md

alexeiva commented:
@impallari this is ready to merge now
@m4rc1e this is ready to push from my repo

m4rc1e committed:
.gitignore: added

m4rc1e commented:
@alexeiva 
The copyright string should look like this:

`Copyright 2010 The Raleway Project Authors (impallari@gmail.com), with Reserved Font Name "Raleway".`

Cheers,
Marc

m4rc1e committed:
source | fonts | OFL: Updated copyright string

alexeiva commented:
Absorbed @m4rc1e's https://github.com/impallari/Raleway/pull/20 in this PR

m4rc1e commented:
@impallari Do you have time to inspect this?

impallari commented:
Yes, later today or tomorrow morning.

thlinard commented:
Hi,
Any update on this?

vladshcherbin commented:
@impallari would be great to have this reviewed and merged.

thlinard commented:
Hi @alexeiva 

According to @m4rc1e this release "can't be [generated as a variable family] with the current Fontmake". https://github.com/google/fonts/pull/1264

m4rc1e commented:
@thlinard the problem is not the font but fontmake. It is having problems generating the avar table.

```
Traceback (most recent call last):
  File "/Users/marc/Documents/googlefonts/variable_fonts/env/bin/fontmake", line 11, in <module>
    sys.exit(main())
  File "/Users/marc/Documents/googlefonts/variable_fonts/env/lib/python2.7/site-packages/fontmake/__main__.py", line 183, in main
    project.run_from_glyphs(glyphs_path, **args)
  File "/Users/marc/Documents/googlefonts/variable_fonts/env/lib/python2.7/site-packages/fontmake/font_project.py", line 378, in run_from_glyphs
    designspace_path, instance_data=instance_data, **kwargs)
  File "/Users/marc/Documents/googlefonts/variable_fonts/env/lib/python2.7/site-packages/fontmake/font_project.py", line 434, in run_from_designspace
    interpolate_layout_from=interpolate_layout_from, **kwargs)
  File "/Users/marc/Documents/googlefonts/variable_fonts/env/lib/python2.7/site-packages/fontmake/font_project.py", line 493, in run_from_ufos
    self.build_variable_font(designspace_path)
  File "/Users/marc/Documents/googlefonts/variable_fonts/env/lib/python2.7/site-packages/fontmake/font_project.py", line 226, in build_variable_font
    font, _, _ = varLib.build(designspace_path, finder)
  File "/Users/marc/Documents/googlefonts/variable_fonts/env/lib/python2.7/site-packages/fontTools/varLib/__init__.py", line 714, in build
    _add_avar(vf, axes)
  File "/Users/marc/Documents/googlefonts/variable_fonts/env/lib/python2.7/site-packages/fontTools/varLib/__init__.py", line 140, in _add_avar
    assert sorted(vals) == vals
AssertionError

```

Once I get the time, I will submit these issues for that tool

thlinard commented:
Thanks for the precision!

davelab6 commented:
@impallari would be great to have this reviewed and merged :)

thlinard commented:
@impallari Hi, any update on this?

thlinard commented:
Hi @m4rc1e: could fontmake do a VF now?

Commit:
Merge pull request #1 from alexeiva/master

Best sources

Commit:
copy in sources from Jill's branch directly to resolve merge conflict

Commit:
clean up repo

Commit:
convert glyphs2ufo

Commit:
try building TTFs from exported .designspace files

Commit:
?

Commit:
update /five.dnom start point

Commit:
try building VFs

Commit:
update build.sh

Commit:
try fixing start point in .glyphs file to match change in .ufo file

Commit:
add all fonts, including otf and vf

Commit:
update build script

Commit:
build all

Commit:
update .glyphs file

Commit:
update designspace files to only use multiples of 100 for weight

Commit:
build.sh

Commit:
remove linebreak from description

Commit:
build

Commit:
rearrange build.sh file

Commit:
build fonts

Commit:
add script to fix TYPOGRAPHIC_SUBFAMILY_NAME

Commit:
update build.sh

Commit:
update build.sh

Commit:
rebuild var fonts

Commit:
rebuild all

Commit:
convert ufo2glyphs

Commit:
update master names in glyphs file

Commit:
convert back glyphs2ufo

Commit:
update Italic glyphs file

Commit:
convert back italic glyphs2ufo

Commit:
update build.sh

Commit:
rebuild

Commit:
remove glyphs files from sources

Commit:
remove unused script

Commit:
add requirement.txt

Commit:
copy in missing glyphs to romans from Impallari sources

Commit:
fix incompatible ligatures

Commit:
remove duplicates

Commit:
copy in /apostrophe

Commit:
copy missing glyphs for italics

Commit:
remove duplicate unicode for /apostrophe

Commit:
copy more glyphs

Commit:
copy glyphs again

Commit:
remove .alt gnames which already exist as, for example .ss01 etc

Commit:
copy missing glyphs into italics

Commit:
fix incompatible

Commit:
clean up repo

Commit:
rebuild var fonts

Commit:
update build script

Commit:
rebuild all

Commit:
ufo2glyphs and glyphs2ufo to fix linebreak in description

Commit:
set FONT_SUBFAMILY_NAME aka styleMapStyleName for Italic

Commit:
layers

Commit:
rebuild

Commit:
Merge pull request #4 from TypeNetwork/fea-ufo-conv2

Fea ufo conv2

Commit:
share reports

Commit:
set missing unicodes

Commit:
build

Commit:
autoUnicodes() for italics

Commit:
add script for reports

Commit:
build all and clean up fonts folder

Commit:
report script

Commit:
fix shifted component in italics

Commit:
build

Commit:
change from using caron to caroncomb component

Commit:
fix component order

Commit:
decompose ['uni01C5', 'uni01C6', 'uni01C4'], regenerate

Commit:
Merge pull request #6 from TypeNetwork/fea-next-master

Fea next master

victoriaalissia committed:
Manually fix Black-Italic accent placement

Commit:
update /Oslashacute components

Commit:
temp comment out build script and report

Commit:
shift acute for /Gacute to match /Oacute

Commit:
uncomment out .sh scripts

Commit:
build all

Commit:
uncomment build scripts

Commit:
rebuild vf

Commit:
update report script

Commit:
rebuild all

Commit:
Merge pull request #7 from TypeNetwork/fea-next-master

Fea next master

Commit:
fix startpoints for /four.dnom in italics

Commit:
rebuild all

Commit:
Merge pull request #9 from TypeNetwork/fea-next-master

Fea next master

m4rc1e committed:
Use styleMapFamilyName and styleMapStyleName

Commit:
Merge pull request #10 from m4rc1e/stylemap

Use styleMapFamilyName and styleMapStyleName

Commit:
outline change requests for hinting prep

/itilde/eight.dnom/gcommaaccent/igrave/ihookabove/utilde/Gacute/00AA/ordmasculine/ordfeminine/zero.lf/one.lf/two.lf/two/three.lf/four.lf/five.lf/six.lf/seven.lf/eight.lf/nine.lf/

Commit:
new build using build.sh

Commit:
fix stroke width 6's

796 = U+2086, sizeinferior
803 = not Unicode, name ‘six.subs’
813 = not Unicode, name ‘six.dnom’
823 = not Unicode, name ‘six.numr’
796 = U+2076, sixsuperior

Commit:
build used build.sh

Commit:
resolve complex component transformations

Commit:
Make new composited chars compatible

Commit:
Merge pull request #14 from TypeNetwork/components

Components

Commit:
Build.sh after composite transformation

Commit:
Merge pull request #15 from TypeNetwork/components

Build.sh after composite transformation

Commit:
Fix to /uni04AB and rebuild

Commit:
Fix variant /esdescender-cy.loclBSH in roman and italic

glyph for Bashkir support

Commit:
Build to include fixes to uni04AB and dup glyphs

mikedug committed:
add VTT source files

Adding VTT sources

mikedug committed:
add vtt sources

add vtt sources

mikedug committed:
Moved VTT Sources to Sources folder

Moved VTT Sources to Sources folder

mikedug committed:
updates to hinting

updates to hinting for Raleway VF Roman and Italic  / April 22nd 2020

mikedug committed:
updates to Hinting

updates to hinting to Raleway and Raleway Italic VF

mikedug committed:
updated hinting

updates to VTT Hinting for Raleway Regular and Italic. continuing to finish glyph set.

mikedug committed:
VTT Hinting updates

VTT Hinting updates for Raleway VF Regular and italic

mikedug committed:
VTT Hint updates

Hinting updates for Roman and Italic Raleway Variable fonts.

mikedug committed:
VTT Hining updates

Updates to Hinting for Raleway Roman and Italic Variable font

mikedug committed:
VTT Hinting update

updates to VTT Hinting for Roman and Italic Raleway Variable fonts

mikedug committed:
Completed VTT Hinting files for Raleway Variable Roman and Italic

Completed, complied and tested, VTT Hinted Variable Raleway font files.

m4rc1e committed:
add hinting to build chain

mikedug committed:
Update VTTSourceRaleway[wght].ttf

update to font hints to fix offset in composite uni04B7

m4rc1e committed:
regenerate fonts

m4rc1e committed:
Remove anchors from non-mark glyphs

m4rc1e committed:
Bump version and regen fonts

Commit:
Merge pull request #17 from mikedug/VTTSources

Completed VTT Hinting sources

Commit:
Merge pull request #18 from m4rc1e/hinted

Add VTT hinting to build chain

alexeiva committed:
Merge pull request #2 from TypeNetwork/master

Completion of the [wght].ttf variable font

thank you @pichotta
